### ISSUE TITLE
1.9: crash in makeAlign, somehow undefined `options`

### ISSUE THREAD

**Issue by wmertens** [prettier/prettier#3413]
When running

```sh
npx prettier --write '{src,lib,plugins}/**/*.{js,jsx,json,md}'
```

I get several of these crashes:

```
[error] src/frontend/components/Pages/ShowRental/RentalList.jsx: TypeError: Cannot read property 'useTabs' of undefined
[error]     at makeAlign (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/index.js:7657:16)
[error]     at makeAlign (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/index.js:7658:11)
[error]     at fits (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/index.js:7700:22)
[error]     at printDocToString$2 (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/index.js:7814:33)
[error]     at formatWithCursor (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/index.js:27649:26)
[error]     at Object.formatWithCursor (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/index.js:27907:12)
[error]     at format (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/bin/prettier.js:20364:21)
[error]     at eachFilename (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/bin/prettier.js:20552:16)
[error]     at ignorer.filter.forEach.filePath (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/bin/prettier.js:20513:9)
[error]     at Array.forEach (<anonymous>)
[error]     at eachFilename (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/bin/prettier.js:20512:8)
[error]     at Object.formatFiles (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/bin/prettier.js:20525:3)
[error]     at Object.run (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/bin/prettier.js:20998:12)
[error]     at Object.<anonymous> (/Users/wmertens/Documents/AeroFS/Projects/meatier/node_modules/prettier/bin/prettier.js:21009:5)
[error]     at Module._compile (module.js:624:30)
[error]     at Object.Module._extensions..js (module.js:635:10)
[error]     at Module.load (module.js:545:32)
[error]     at tryModuleLoad (module.js:508:12)
[error]     at Function.Module._load (module.js:500:3)
[error]     at Function.Module.runMain (module.js:665:10)
[error]     at findNodeScript.then.existing (/nix/store/01kx9kbbmbm5gvb0rm2k02nkhrzyksrb-nodejs-8.6.0/lib/node_modules/npm/node_modules/libnpx/index.js:260:14)
[error]     at <anonymous>
```

There are many files that it formats without crashing. Not sure what triggers it.

my configuration is part of `package.json`:

```json
{
  "prettier": {
    "useTabs": true,
    "semi": false,
    "singleQuote": true,
    "trailingComma": "es5",
    "bracketSpacing": false
  }
}
```
Node 8.6.0

**Comment by duailibe** [comment]
Thanks for the report! This was fixed in #3401 and will be release in 1.9.1.

**Comment by duailibe** [comment]
The fix was released in 1.9.1!
